### DISCUSSION TITLE
MQTT server with four characters Password

### DISCUSSION THREAD

**Discussion by fritserasmus** [arendst/Tasmota#18141] [Q&A] [answered]
@arendst ,
I have been flashing SOnOffs since ver 
Program Version | 5.14.0
2018-05-15T15:29:54
2_3_0/1.5.3(aec24ac9)

I have MQTT connections in the region of 60 connections and all set up to use a 4-character long password.

Some are arduinos hardly accessible to reload the software and the new Tasmota requires a 5-character PW .

How can I go about setting p Tasmota cureent version web install to accept a password for my MQTT server to be 4 characters long.
All my devices are in my home network, not exposed to the WWW.

Your help will really be appreciated

**Comment by barbudor** [comment, marked as answer]
The new software requires a 5 character password only if you flash from scratch and enter the password in the configuration web UI

If case of upgrade you will not be asked to set a new password
Also the password length is not enforced in the console command `MqttPassword`

**Reply by fritserasmus** [reply]
Thanks for your reply @barbudor ,

I did not upgrade an existing NODE-MCU, I added one to my pool of sensors.
I tried to add the MQTT via the console, but that also was not successful.

**Reply by barbudor** [reply]
Then your doing it wrong
I have no problem entering
```
MqttPassword abc
```
